Packs/CommonPlaybooks/Playbooks/playbook-Search_For_Hash_In_Sandbox_-_Generic.yml

```diff
@@ -1,719 +1,744 @@
+    view: |-
+      {
+        "position": {
+          "x": -40,
+          "y": 735
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "23":
+    id: "23"
+    taskid: f1e1594d-5b8b-4414-8efb-2682ee0bb1a2
+    type: regular
+    task:
+      id: f1e1594d-5b8b-4414-8efb-2682ee0bb1a2
+      version: -1
+      name: Get Wildfire PDF report
+      description: Gets a PDF report of search results for hashes in the WildFire
+        sandbox.
+      script: '|||wildfire-report'
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "25"
+    scriptarguments:
+      format:
+        simple: pdf
+      sha256:
+        simple: ${inputs.FileSha256}
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 950,
+          "y": 490
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "24":
+    id: "24"
+    taskid: 93e1ecdb-1db8-4189-8aea-f455c87ebab2
+    type: regular
+    task:
+      id: 93e1ecdb-1db8-4189-8aea-f455c87ebab2
+      version: -1
+      name: Rasterize PDF
+      description: Converts a PDF file to an image file.
+      script: '|||rasterize-pdf'
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "19"
+    scriptarguments:
+      EntryID:
+        complex:
+          root: InfoFile
+          filters:
+          - - operator: isEqualString
+              left:
+                value:
+                  simple: InfoFile.Extension
+                iscontext: true
+              right:
+                value:
+                  simple: pdf
+              ignorecase: true
+          accessor: EntryID
+          transformers:
+          - operator: uniq
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 1190,
+          "y": 830
+        }
+      }
+    note: false
+    evidencedata:
+      description:
+        simple: Wildfire report
+      customfields: {}
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "25":
+    id: "25"
+    taskid: 41001a52-b698-414c-8561-a9f8a3c2d620
+    type: condition
+    task:
+      id: 41001a52-b698-414c-8561-a9f8a3c2d620
+      version: -1
+      name: Is there a report to rasterize?
+      type: condition
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#default#':
+      - "19"
+      "yes":
+      - "24"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: isEqualString
+          left:
+            value:
+              simple: InfoFile.Extension
```

Comment:
Is the key correct? usually we are using the `File` context path

Reply:
The Wildfire reports are also written here in context
Makes it easier to find in case you have another un related PDF in context